packages/core/src/tools/utils.ts

```diff
@@ -564,3 +564,9 @@ export function combine(...sources: any[]): unknown {
 }
 
 export type TimeoutId = ReturnType<typeof setTimeout>
+
+export function removeDuplicates<T>(array: T[]) {
+  const deduplicated: T[] = []
+  new Set(array).forEach((item) => deduplicated.push(item))
+  return deduplicated
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏
```suggestion
  return Array.from(new Set(array))
```

Reply:
Nice ! I though we could not use `Array.from` because of IE.